docs/reference/glossary.rst

```diff
@@ -71,7 +71,7 @@ Glossary
     APT
         Abbreviation for *Advanced Package Manager*. 
         
-        See: :doc:`/reference/apt`
+        See: `Advanced Packaging Tool (Ubuntu Server documentation) <https://ubuntu.com/server/docs/package-management#advanced-packaging-tool>`_
```

Comment:
Linkcheck fails to find the heading for this link:
> (reference/glossary: line   74) broken    https://ubuntu.com/server/docs/package-management#advanced-packaging-tool - Anchor 'advanced-packaging-tool' not found

The link is correct (because I copied it from the existing page) and I can also see that it is there:
```
$curl -o - -H "Accept: text/html" -X GET https://ubuntu.com/server/docs/package-management | grep advanced-packaging-tool

<h2><a class="p-link--anchor-heading" href="#advanced-packaging-tool" name="advanced-packaging-tool">Advanced Packaging Tool</a></h2>
```
So it is not a JavaScript issue like for manpages.ubuntu.com

My theory is that linkcheck only looks for links with `<a id="NAME">` and does not recognize links with `<a name="NAME">`, which is kind of understandable when you consider that the `name` attribute is depricated (see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#name). 

Minor rant:
Still, the `name` attribute was a valid attribute and linkcheck should expect that there are websites still using it and it is annoying that linkcheck does not recognize this and I have to do all the stuff above, which is effectively a huge waste of time.

Reply:
There is a fix for this issue on the way, so it shouldn't blocking this PR